### ISSUE TITLE
Whether Chinese voice is supported?

### ISSUE THREAD

**Issue by pango99** [zhang-haojie/LetsTalk#2]
Whether Chinese voice is supported?

**Comment by zhang-haojie** [comment]
Yes, you can visit our [project page](https://zhang-haojie.github.io/project-pages/letstalk.html) to see examples.

**Comment by suimuc** [comment]
Hello, is any other language supported? I'm eager to know whether the audios in the dataset are all in English, or if they include any other languages. 
l wish you would answer my guestion. Thank you very much.

**Comment by suimuc** [comment]
@zhang-haojie

**Comment by zhang-haojie** [comment]
Sorry, we haven't tested all languages. As far as we know, the train dataset is primarily English audio, but we still achieve good results with Chinese audio. Therefore, we speculate that our model has achieved a certain level of zero-shot generalization in audio.

**Comment by suimuc** [comment]
> Sorry, we haven't tested all languages. As far as we know, the train dataset is primarily English audio, but we still achieve good results with Chinese audio. Therefore, we speculate that our model has achieved a certain level of zero-shot generalization in audio.

Thank you very much!
